src/tests/simulation/src/cluster.rs

```diff
@@ -438,7 +442,14 @@ impl Cluster {
             tracing::info!("kill {name}");
             madsim::runtime::Handle::current().kill(name);
 
-            let t = rand::thread_rng().gen_range(Duration::from_secs(0)..Duration::from_secs(1));
+            let mut t =
+                rand::thread_rng().gen_range(Duration::from_secs(0)..Duration::from_secs(1));
+            // has a small chance to restart after a long time
+            // so that the node is expired and removed from the cluster
```

Comment:
I guess the node will be assigned a new worker id?

Reply:
Yes